client/src/api.ts

```diff
@@ -69,6 +93,21 @@ async function secureFetchNoResponse(input: RequestInfo, init?: RequestInit): Pr
   }
 }
 
+async function secureFetch(input: RequestInfo, init?: RequestInit): Promise<void> {
```

Comment:
Optionally we can add a Generic type that would allows us to know what `secureFetch` will return.

Code example:
```typescript
async function secureFetch<A>(input: RequestInfo, init?: RequestInit): Promise<A> {
```

Demo:
```typescript
type Monkey = {
  name: string,
};

const monkey = secureFetch<Monkey>('https://monkeys.io/api/v1/donkey-kong');
```

Reply:
Done, thanks!